sample-apps/radiology/lib/transforms/transforms.py

```diff
@@ -505,3 +506,35 @@ def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> Dict[Hashable, N
             if d.get(cache_key) is None:
                 d[cache_key] = copy.deepcopy(d[key])
         return d
+
+
+class OrientationGuidanceMultipleLabelDeepEditd(Transform):
+    def __init__(self, label_names=None):
+        """
+        Convert the guidance to the RAS orientation
+        """
+        self.label_names = label_names
+
+    def transform_points(self, point, affine):
+        """transform point to the coordinates of the transformed image
+        point: numpy array [bs, N, 3]
+        """
+        bs, N = point.shape[:2]
+        point = np.concatenate((point, np.ones((bs, N, 1))), axis=-1)
+        point = rearrange(point, "b n d -> d (b n)")
+        point = affine @ point
+        point = rearrange(point, "d (b n)-> b n d", b=bs)[:, :, :3]
+        return point
+
+    def __call__(self, data):
+        d: Dict = dict(data)
+        for key_label, val_label in self.label_names.items():
+            points = d.get(key_label, [])
+            if len(points) < 1:
+                continue
+            reoriented_points = self.transform_points(
+                np.array(points)[None],
+                np.linalg.inv(d["image_meta_dict"]["affine"].numpy()) @ d["image_meta_dict"]["original_affine"],
```

Comment:
Hi @diazandr3s , could you switch to use `d["image"].meta["affine"]` and `d["image"].meta["original_affine"]` ?

`"image_meta_dict"` seems outdated and may not update after transforms. The following is my local test code (on MONAI 1.2.0):

```
from monai.transforms import LoadImaged, Orientationd, Compose, EnsureChannelFirstd

image_path = "/workspace/data/medical/Task09_Spleen/imagesTr/spleen_10.nii.gz"

transforms = Compose([
    LoadImaged(keys="image"),
    EnsureChannelFirstd(keys="image"),
    Orientationd(keys="image", axcodes="LPI"),
])

image = transforms({"image": image_path})

print(image["image_meta_dict"]["original_affine"])
print(image["image_meta_dict"]["affine"])
print(image["image"].meta["affine"])
```

<img width="1070" alt="Screenshot 2023-09-05 at 7 33 43 PM" src="https://github.com/Project-MONAI/MONAILabel/assets/68361391/c0d4973b-c678-4408-a5ea-b4b1660e3819">

BTW, Hi @Nic-Ma @wyli @SachidanandAlle @diazandr3s we may also need to other deepedit related code in MONAI core and MONAILabel if using `"image_meta_dict"`, what do you think?

Reply:
For example, this transform:
https://github.com/Project-MONAI/MONAI/blob/8e77e03105c745936d4ffd98ed536f2bdc132b48/monai/apps/deepedit/transforms.py#L701

Reply:
@diazandr3s lets get this updated/fixed as per the recommendations.  Let me know if you need any help

Reply:
Many thanks for checking this, @yiheng-wang-nv.

I just updated this transform following your suggestions.

The reason for the labels to be a dictionary is that, for some transforms, we need to keep track of the label indexes as well:

`{'spleen': 1, 'right kidney': 2, 'left kidney': 3, 'liver': 6, 'stomach': 7, 'aorta': 8, 'inferior vena cava': 9, 'background': 0}`

This is especially important when the indexes are not contiguous.

Reply:
Thanks for the explain @diazandr3s . I think this PR is fine, and for the suggestions of update other transforms, let me create a separate ticket in MONAI core thus I can resolve this comment.